website/docs/tutorials/07-testing/02-cheatcodes/max-examples.md

```diff
@@ -0,0 +1,43 @@
+# `max_examples`
+
+```python
+def max_examples(max_examples: int):
+```
+
+Sets the maximum value of examples to explore by the fuzzer.
+Once this many satisfying input cases have been evaluated without finding any counter-example,
+fuzzer falsification will terminate.
+
+:::warning
+This cheatcode is only available in [setup hooks](../README.md#__setup__).
+:::
+
+Protostar defaults to 100 examples.
+This default value is chosen to suit a workflow where the test will be part of a suite that
+is regularly executed locally or on a CI server,
+balancing total running time against the chance of missing a bug.
+
+If you are writing one-off tests, running tens of thousands of examples is quite reasonable
+as Protostar's fuzzer may miss uncommon bugs with default settings.
+
+This cheatcode has no effects on standard test cases.
+
+```cairo title="Example"
```

Comment:
Can we move example up, above warning?

It is more important and right now it is not clear what it refers to

Reply:
I moved it just under the warning to match style of assume/reject docs. Good catch!